File: app/src/main/java/com/wegielek/signalychinese/database/Radicals.java
package com.wegielek.signalychinese.database;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Radicals {
    @PrimaryKey
    public int uid;

    @NonNull
    @ColumnInfo(name = "section")
    public String section = "";

    @NonNull
    @ColumnInfo(name = "radicals")
    public String radicals = "";
}

File: app/src/main/java/com/wegielek/signalychinese/database/Radicals.kt
package com.wegielek.signalychinese.database

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Radicals {
    @PrimaryKey(autoGenerate = true)
    var uid = 0

    @ColumnInfo(name = "section")
    var section = ""

    @ColumnInfo(name = "radicals")
    var radicals = ""
}